backend/ibutsu_server/openapi/openapi.yaml

```diff
@@ -2388,6 +2661,33 @@ components:
           description: The date this artifact was uploaded
           type: string
       type: object
+    Portal:
+      example:
+        id: fe5ad2dc-330a-11ef-9036-12b95372ee33
+        name: my-portal
+        title: My portal
+        owner_id: 07a776ba-330b-11ef-9338-12b95372ee33
+      properties:
```

Comment:
this one is missing the required entry

Reply:
agreed - as are all the other schema here. Not sure why, conceptually there are absolutely required fields in these objects.

Will take a second look, might be an easy API spec "upgrade".

Reply:
it may be a god idea to do a typespec import/fixup as a followup